blog/blog.html

```diff
@@ -340,6 +311,7 @@ <h3>KyootyBella</h3>
     <!-- Java Script
    ================================================== -->
 
+    <script src="../scripts/index.js"></script>
```

Comment:
I'm placing your index.js at the bottom of the template. It doesn't mean placing it at the top is a problem per se, but read through the following and you'll understand why:

---

The best practice is to put JavaScript `<script>` tags just before the closing `</body>` tag rather than in the `<head>` section of your HTML.

The reason for this is that HTML loads from top to bottom. The head loads first, then the body, and then everything inside the body. If we put our JavaScript in the head section, the entire JavaScript will load before loading any of the HTML, which could cause a few problems :

If you have code in your JavaScript that alters HTML as soon as the JavaScript code loads, there won’t actually be any HTML elements available for it to affect yet, so it will seem as though the JavaScript code isn’t working, and you may get errors.
If you have a lot of JavaScript, it can visibly slow the loading of your page because it loads all of the JavaScript before it loads any of the HTML.
When you place your JavaScript at the bottom of your HTML body, it gives the HTML time to load before any of the JavaScript loads, which can prevent errors, and speed up website response time.